manpages/lumina-config.8

```diff
@@ -1 +1,25 @@
-Dummy file
+.Dd November 3, 2017
+.Dt LUMINA-CONFIG 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-config
+.Nd used to modify different desktop settings.
+
+.Sh SYNOPSIS
+.Nm
+
+.Sh DESCRIPTION
+.Nm
+opens the graphical utility to modify many different desktop settings.
```

Comment:
"many different" is unnecessary. "desktop settings" implies there are different options.